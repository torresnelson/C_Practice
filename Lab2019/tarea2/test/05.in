# prueba de pertenece, longitud, cambiar_todos, concatenar
pertenece 1
longitud
cambiar_todos 5 6
concatenar .
concatenar (9,i) .
insertar_al_final (5,e)
concatenar .
concatenar (9,i) .
cambiar_todos 5 6
imprimir_cadena
cambiar_todos 6 5
longitud
pertenece 1
pertenece 5
final_cadena
insertar_segmento_despues (2,a) (5,b) (1,c) (4,d) .
concatenar .
concatenar (9,i) .
concatenar (9,i) (8,h) (7,g) (6,f) .
longitud
cambiar_todos 6 3
pertenece 5
pertenece 1
pertenece 2
pertenece 4
pertenece 3
pertenece 6
cambiar_todos 5 6
pertenece 5
pertenece 6
imprimir_cadena
Fin