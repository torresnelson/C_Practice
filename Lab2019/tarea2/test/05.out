1>#  prueba de pertenece, longitud, cambiar_todos, concatenar.
2>1 no pertenece a cad.
3>Longitud: 0
4>Cambiados.
5>
6>(9,i)
7>Insertado al final.
8>(5,e)
9>(5,e)(9,i)
10>Cambiados.
11>(6,e)
12>Cambiados.
13>Longitud: 1
14>1 no pertenece a cad.
15>5 pertenece a cad.
16>loc al final.
17>Segmento insertado despuÃ©s de loc.
18>(5,e)(2,a)(5,b)(1,c)(4,d)
19>(5,e)(2,a)(5,b)(1,c)(4,d)(9,i)
20>(5,e)(2,a)(5,b)(1,c)(4,d)(9,i)(8,h)(7,g)(6,f)
21>Longitud: 5
22>Cambiados.
23>5 pertenece a cad.
24>1 pertenece a cad.
25>2 pertenece a cad.
26>4 pertenece a cad.
27>3 no pertenece a cad.
28>6 no pertenece a cad.
29>Cambiados.
30>5 no pertenece a cad.
31>6 pertenece a cad.
32>(6,e)(2,a)(6,b)(1,c)(4,d)
33>Fin.
